packages/react-aria-components/docs/Group.mdx

```diff
@@ -0,0 +1,274 @@
+    border-radius: 0 6px 6px 0;
+    align-self: stretch;
+  }
+}
+
+.react-aria-Button {
+  --border-color: var(--spectrum-alias-border-color);
+  --border-color-pressed: var(--spectrum-alias-border-color-down);
+  --border-color-disabled: var(--spectrum-alias-border-color-disabled);
+  --background-color: var(--spectrum-global-color-gray-50);
+  --background-color-pressed: var(--spectrum-global-color-gray-100);
+  --text-color: var(--spectrum-alias-text-color);
+  --text-color-disabled: var(--spectrum-alias-text-color-disabled);
+  --focus-ring-color: slateblue;
+
+  color: var(--text-color);
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  appearance: none;
+  vertical-align: middle;
+  font-size: 1rem;
+  text-align: center;
+  margin: 0;
+  outline: none;
+  padding: 6px 10px;
+  text-decoration: none;
+
+  &[data-pressed] {
+    box-shadow: inset 0 1px 2px rgb(0 0 0 / 0.1);
+    background: var(--background-color-pressed);
+    border-color: var(--border-color-pressed);
+  }
+
+  &[data-focus-visible] {
+    border-color: var(--focus-ring-color);
+    box-shadow: 0 0 0 1px var(--focus-ring-color);
+  }
+}
+
+@media (forced-colors: active) {
+  .react-aria-TextField {
+    --field-border: ButtonBorder;
+    --field-border-disabled: GrayText;
+    --field-background: Field;
+    --text-color: FieldText;
+    --text-color-disabled: GrayText;
+    --focus-ring-color: Highlight;
+    --invalid-color: LinkText;
+  }
+}
+```
+
+</details>
+
+## Features
+
+A group can be created with a `<div role="group">` or via the HTML [&lt;fieldset&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) element. The `Group` component supports additional UI states, and can be used standalone or as part of a larger pattern such as [NumberField](NumberField.html) or [DatePicker](Datepicker.html).
+
+* **Styleable** – Hover, keyboard focus, disabled, and invalid states are provided for easy styling. These states only apply when interacting with an appropriate input device, unlike CSS pseudo classes.
+* **Accessible** – Implemented using the ARIA "group" role by default, with optional support for the "region" landmark role.
+
+## Anatomy
+
+A group consists of a container element for a set of semantically related UI controls. It supports states such as hover, focus within, and disabled, which are useful to style visually adjoined children.
+
+```tsx
+import {Group} from 'react-aria-components';
+
+<Group>
+  {/* ... */}
+</Group>
+```
+
+## Accessibility
+
+### Labeling
+
+Group accepts the `aria-label` and `aria-labelledby` attributes to provide an accessible label to the group as a whole. This is read by assistive technology when navigating into the group from outside. When the labels of each child element of the group do not provide sufficient context on their own, the group should receive an additional label.
+
+```tsx example
+<span id="label-id">Social security number</span>
```

Comment:
lets not make an example where people enter social security numbers

phone might be fine, that's used as an example in many places, so it's more expected
also, many countries don't have social security numbers

maybe lat/long or x/y/z or...

Reply:
is it problematic for some reason? not like we're submitting them or anything.

Reply:
no, just feels weird/sketchy

Reply:
I would probably lean towards changing it just to avoid improperly implementing that field (USWDS says [not to separate it into three separate fields](https://designsystem.digital.gov/patterns/create-a-user-profile/social-security-number/)), and being more easily understood across countries. Good to approve after an update.

Reply:
hmm fwiw, I based this on an example from the WCAG docs. https://www.w3.org/WAI/WCAG22/working-examples/aria-grouping-multipart-fields-ssn/

Reply:
What should I change it to? phone number also wouldn't work because different countries have different formats. some generic "Account number"? use the same expiration date example as at the bottom of the page?

Reply:
i think a generic "account number" could work or maybe a generic "serial number". you could also do [zip codes](https://pe.usps.com/archive/html/dmmarchive20030810/A010.htm#:~:text=The%20most%20complete%20ZIP%20Code,a%20hyphen%2C%20and%20four%20digits.) but that might be too US centric, or MAC addresses

idk im just shooting out some ideas